app/controllers/insured/consumer_roles_controller.rb

```diff
@@ -293,6 +293,9 @@ def help_paying_coverage
       set_admin_bookmark_url
       @transaction_id = params[:id]
       @shop_coverage_result ||= params[:shop_coverage_result]
+
+      draft_application = @person.primary_family&.draft_financial_assistance_application if EnrollRegistry.feature_enabled?(:draft_application_after_ridp)
```

Comment:
we are not seding here the current enrollment year, by default draft_financial_assistance_application will chose current year, but what happens when we are past open enrollment ? as of now, it should be redirected to 2023 right